src/snowcli/cli/nativeapp/manager.py

```diff
@@ -152,7 +152,11 @@ def stage_schema(self) -> Optional[str]:
         return extract_schema(self.stage_fqn)
 
     @cached_property
-    def warehouse(self) -> Optional[str]:
+    def package_warehouse(self) -> Optional[str]:
+        return self.definition.get("package", {}).get("warehouse", self._conn.warehouse)
```

Comment:
Either both `package_warehouse` and `application_warehouse` should default to `self._conn.warehouse`, or both should default to `None`. I'm for the latter, personally

Reply:
Resolved offline.